src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -270,8 +258,6 @@ private function getAutowiredReference(TypedReference $reference, $deprecationMe
         if (isset($this->autowired[$type])) {
             return $this->autowired[$type] ? new TypedReference($this->autowired[$type], $type) : null;
         }
-
-        return $this->createAutowiredDefinition($type);
```

Comment:
@dunglas @nicolas-grekas @GuilhemN Can you confirm that this is the proper fix here?

Reply:
LGTM